### ISSUE TITLE
URLs are treated as unique security origins.

### ISSUE THREAD

**Issue by Onenotmind** [w3c/webvtt#484]
why chrom can't load vtt file

**Comment by gkatsev** [comment]
Is this still an issue? Do you have an example to try out?

**Comment by Dggamer007k7** [comment]
Hello, i'm going through a similar problem as the dude above said, im trying to use a vtt file on a html page ( both the vtt, the video and the page are in the same folder, after using Edge and Opera gx it doesn't seem to work 
" unsafe attempt to load URL XXXXXX from frame with URL XXXXXX 'file:' URLs are treated as unique security origins. "

**Comment by Dggamer007k7** [comment]
maybe... perhaps... because they're in a flash drive and not in the computer itself but i don't know if that's related

**Comment by Dggamer007k7** [comment]
Nope, it doesn't work even if the files are in the pc, same error
